runtime/queries/elixir/indents.scm

```diff
@@ -0,0 +1,13 @@
+[
+  (after_block)
+  (anonymous_function)
+  (catch_block)
+  (do_block)
+  (else_block)
+  (rescue_block)
+  "->"
```

Comment:
```suggestion
  (stab_clause)
```

The `->` won't work on its own because it doesn't have any children nodes for the indentation to apply to

Reply:
Interestingly, it did work when i tried it, however your suggestion is better.